Source/stream.ts

```diff
@@ -278,6 +278,15 @@ const FOX_PLUS: StreamService = {
   ]
 };
 
+const FRIDAY: StreamService = {
+  name: 'friDay Video',
+  rules: [
+    'DOMAIN-SUFFIX,fetnet.net',
+    'DOMAIN-SUFFIX,friday.tw',
+    'DOMAIN-REGEX,^aka\d+p\.akamaized\.net'
```

Comment:
No, we only accept `DOMAIN-WILDCARD` here. The `DOMAIN-REGEX` would be automatically output through the building script.

Reply:
Also, last time I checked, there is no geo restriction being put on `aka40p.akamaized.net` or any other similar domains.

Please provide evidence that this domain can only be accessed through Taiwan IPs.